src/www/ui/api/documentation/openapi.yaml

```diff
@@ -3618,12 +3744,12 @@ paths:
       description: >
         Updates the admin configuration data.
       requestBody:
-        description: Key and value to update.
+        description: Array of keys and values to update.
         required: true
         content:
           application/json:
             schema:
-              $ref: '#/components/schemas/SetCustomiseInfo'
+              $ref: '#/components/schemas/ConfigUpdateRequest'
```

Comment:
This will break the backwards compatibility of the endpoint. Please keep the changes only for API V2.

Reply:
Done, Migrated the `/customize` endpoint changes to V2 only.